### PR TITLE
Fix config error

### PR DESCRIPTION
**Pull request by huwshimi**
Fixes: #2994. Fixes: #2991.

### CODE REVIEW

File: jujugui/static/gui/src/app/extensions/app-cookies-extension.js [hunk 1]
```diff
@@ -143,6 +143,5 @@ YUI.add('app-cookies-extension', function(Y) {
 
 }, '0.1.0', {
   requires: [
-    'cookie-notification'
```
Comment (huwshimi) [inline jujugui/static/gui/src/app/extensions/app-cookies-extension.js]: Drive-by for #2991.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7788/

File: jujugui/static/gui/src/app/components/inspector/config/test-config.js [hunk 2]
```diff
@@ -99,6 +99,129 @@ describe('Configuration', function() {
         config={option2key} />);
   });
 
+  it('displays the save button when there are changes', function() {
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/inspector/config/test-config.js]: Great tests thanks

**Comment by hatched** [comment]
Thanks @huwshimi :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge
